### DISCUSSION TITLE
1.5 - Status (UE-5.3)

### DISCUSSION THREAD

**Discussion by muit** [PipeRift/SaveExtension#92] [General]
## What's Changed (and whats to be done)
- [x] https://github.com/PipeRift/SaveExtension/pull/82 by @muit
- [x] https://github.com/PipeRift/SaveExtension/pull/88 by @muit
- [x] https://github.com/PipeRift/SaveExtension/pull/89 by @muit
- [x] https://github.com/PipeRift/SaveExtension/pull/91 by @muit
- [x] Thumbnails are saved inside the save file
- [x] #94 by @muit 
- [ ] Fix serialization warning of texture in rendering thread
- [ ] Give more control on the destination of the save (to memory or to file)
- [ ] Provide target level on GetLevelFilter
- [x] #93 by @muit
- [ ] Use unreal's native save system interface for console support

**Full Changelog**: https://github.com/PipeRift/SaveExtension/compare/1.4...1.5.p1

<hr /><em>This discussion was created from the release <a href='https://github.com/PipeRift/SaveExtension/releases/tag/1.5.b1'>1.5 - Beta 1 (UE-5.3)</a>.</em>

**Comment by Showd541** [comment]
This doesn't work for 5.0.3, am I doing something wrong or is that intentional? Not being needy but I'm curious.

**Reply by muit** [reply]
This version starts at 5.3 sadly :(

**Reply by Showd541** [reply]
Dang, wish there was a version for 5.0

**Reply by muit** [reply]
If you are a coder, feel free to backport it :) unfortunately I cant support many versions since I have very limited time

**Reply by Showd541** [reply]
Fair enough

**Reply by AnasTheDev** [reply]
I used the older version of the plugin and it worked just fine, this one, I got it to appear on my plugin and I activated it but I do not see the nodes, what happened?

**Reply by AnasTheDev** [reply]
seems that the nodes were simply changed, but the plugin is causing whenever I save, what is causing this?

**Reply by sweetswe20** [reply]
Does this work for 4.27 by any chance, or is there any other release that works for 4.27? A bit desperate for it. Thanks.
